tests/integration/cli/test_plugins.py

```diff
@@ -25,8 +25,17 @@ def wrapper(f):
 
 @pytest.fixture
 def installed_plugin(runner, ape_cli):
-    # TODO: Figure out how to install plugin prior to test **that actually works**.
-    runner.invoke(ape_cli, ["plugins", "install", TEST_PLUGIN_NAME])
+    # TODO: Figure out how to correctly modify site packages mid-test run.
+    #  The installed plugin does not show in `list` command but does at end of tests.
+    is_already_installed = TEST_PLUGIN_NAME in runner.invoke(ape_cli, ["plugins", "list"]).output
+
```

Comment:
I haven't used this myself yet but I did find this when looking into changing the installation path for packages:

https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUSERBASE

Reply:
so we can point that at tempdir and that is much safer, because then we never adjusting user's actual site packages

Reply:
it seems to not work when running mid python session already

Reply:
TBH this fixture isn't working anyway, so I made it assume it is installed already for now.